apps/server/src/shared/domain/service/task-copy.service.ts

```diff
@@ -25,22 +25,7 @@ export class TaskCopyService {
 			lesson: params.destinationLesson,
 		});
 
-		const elements = [...this.defaultTaskStatusElements(), ...this.copyTaskFiles(params.originalTask)];
-
-		const status: CopyStatus = {
-			title: copy.name,
-			type: CopyElementType.TASK,
-			status: this.copyHelperService.deriveStatusFromElements(elements),
-			copyEntity: copy,
-			originalEntity: params.originalTask,
-			elements,
-		};
-
-		return status;
-	}
-
-	private defaultTaskStatusElements(): CopyStatus[] {
-		return [
+		const elements = [
```

Comment:
was all the stuff removed by accident? does it have anything to do with courseGroups?